### PR TITLE
[DOCS] Intro to Elastic Security 

### PR DESCRIPTION
**Pull request by jmikell821**
[Elastic Security Overview Preview link. ](https://security-docs_85.docs-preview.app.elstc.co/guide/en/security/master/es-overview.html)

### CODE REVIEW

**Comment by bradenlpreston** [comment]
@jmikell821 - can ensure Jae Lee and Mark Settle can access this? I wasnt able to add them.

They should review the content.

**Comment by bradenlpreston** [comment]
My gut tells me that this should be focused on the new endpoint security offering and less Endgame.

- Advanced endpoint detection, prevention, and response for continuous protection across MITRE ATT&CK®

[BP] - today we dont have endpoint response or what would be considered "Advanced Prevention"

- A powerful rules engine to collect events and generate external alerts for system health monitoring

[BP] - A powerful detection engine to identify attacks and system misconfiguration

- Visualized event analysis that shows a timeline of events that occurred before and after an attempted attack

[BP] - Powerful visualizations to investigate attacks

- Accelerated response with embedded case management and automated actions

- Detection of signatureless attacks with machine learning and technique-based methods
- Prebuilt anomaly detection jobs and detection rules
[BP] - can we combine these 2 bullets?

- Creation of exception lists to minimize false positives
[BP] - Do we think this is necessary? I feel this is a bit of a given (or should be)

File: docs/getting-started/es-overview.asciidoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[[es-overview]]
```
Comment (benskelker) [inline docs/getting-started/es-overview.asciidoc:1]:
```suggestion
[[es-overview]]
[chapter]
```

File: docs/siem-apis.asciidoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [role="xpack"]
 [[security-apis]]
-= Elastic Security APIs
+== Elastic Security APIs
```
Comment (benskelker) [inline docs/siem-apis.asciidoc:3]:
```suggestion
= Elastic Security APIs
```

**Comment by gtback** [comment]
Thanks, @benskelker. I tried a bunch of things to get this to work, but figured you'd know best what the "right" one was.

**Comment by jmikell821** [comment]
> * Prebuilt anomaly detection jobs and detection rules
>   [BP] - can we combine these 2 bullets?

Hey @bradenlpreston - thanks for the feedback. Which two bullets should I combine? I couldn't tell in the comments.

**Comment by bradenlpreston** [comment]
@jmikell821 - these two

- Detection of signatureless attacks with machine learning and technique-based methods
- 
- Prebuilt anomaly detection jobs and detection rules

**Pull request merged**